static/app.js

```
@@ -0,0 +1,228 @@
+
+    const items = popup.children;
+    if (event.key === 'ArrowDown') {
+        selectedIndex = (selectedIndex + 1) % items.length; // Move down
+    } else if (event.key === 'ArrowUp') {
+        selectedIndex = (selectedIndex - 1 + items.length) % items.length; // Move up
+    } else if (event.key === 'Enter') {
+        if (selectedIndex >= 0 && selectedIndex < items.length) {
+            const selectedItem = items[selectedIndex].textContent;
+            let user_query = document.getElementById('messageInput').value 
+            console.log(user_query)
+            user_query = user_query.trim().split(' ').slice(0, -1).join(' ') + ' ' + selectedItem;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)